terraform/deployments/govuk-publishing-platform/origins_certificates.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_acm_certificate" "public_north_virginia" {
+  provider    = aws.us_east_1
```

Comment:
yes, some elements of Cloudfront can only be terrafrom in us_east_1. See e.g. ref: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cnames-and-https-requirements.html

Reply:
> If you want to require HTTPS between viewers and CloudFront, you must change the AWS Region to US East (N. Virginia) in the AWS Certificate Manager console before you request or import a certificate.

Wow. Does that mean the certificate manager requests occur in `us_east_1`, but the resources (certs, route53, ALBs etc) are in `eu_west_1`?

Reply:
the certs, waf (all resources with provider=aws.us_east_1), etc are in us_east_1 because they can't be elsewhere due to their link with cloudfront. Other resources like ALBs are in our default region.

Reply:
That is surprising. Does this mean that all requests to GOV.UK will go through `us_east_1` (the flakiest AWS region) and then to `eu_west_1`? I wonder what impact that will have on site reliability.

Reply:
If I understand correctly, CloudFront is a global service but has some elements of its management plane in us-east-1. The data plane (i.e. user traffic etc.) shouldn't depend on or necessarily traverse us-east-1. I'm not sure about the control plane 🤔